clients/venice-samza/src/main/java/com/linkedin/venice/pushmonitor/RouterBasedHybridStoreQuotaMonitor.java

```diff
@@ -124,6 +124,10 @@ protected void checkStatus() throws ExecutionException, InterruptedException, Ti
       // Get hybrid store quota status
       CompletableFuture<TransportClientResponse> responseFuture = transportClient.get(requestPath);
       TransportClientResponse response = responseFuture.get(POLL_TIMEOUT_MS, TimeUnit.MILLISECONDS);
+      if (response == null) {
+        LOGGER.error("Router was not able to get hybrid quota status! Received null response!");
+        return;
+      }
```

Comment:
What is the behavior with early bailout? Would we skip any quota enforcement or block the push or let the caller decide?

Reply:
The current behavior (sans this change even) on any error for failing to ascertain the quota enforcement state is to do nothing, it will allow the writer to continue in the current state.